File: Livros base/fundamentos da programação 3ª edicao/lista de exercicios - condicionais/ER-4.c
#include <stdio.h>

void classificao(int a, int b, int c);

void classificao(int a, int b, int c){
	if(a < b && a < c){
		if(b < c)
			printf("%d - %d - %d", a, b, c);
		else
			printf("%d - %d - %d", a, c, b);
	}if(b < a && b < c){
		if(a < c)
			printf("%d - %d - %d", b, a, c);
		else
			printf("%d - %d - %d", b, c, a);
	}else if(c < a && c < b){
		if(a < b)
			printf("%d - %d - %d", c, a, b);
		else
			printf("%d - %d - %d", c, b, a);
	}
}

void main(){
	int a, b, c;
	printf("escreva o numero a: ");
	scanf("%d*c", &a);
	printf("escreva o numero b: ");
	scanf("%d*c", &b);
	printf("escreva o numero c: ");
	scanf("%d*c", &c);
	printf("\n");
	classificao(a, b, c);
}

File: Livros base/fundamentos da programação 3ª edicao/lista de exercicios - condicionais/ERCondicionais/src/ercondicionais/ER4.java
package ercondicionais;
import java.util.*;

public class ER4 {
    public int n1, n2, n3;
    Scanner input = new Scanner(System.in);
    public void er4(){
        System.out.println("Escreva o primeiro numero: ");
        n1 = input.nextInt();
        System.out.println("Escreva o segundo numero: ");
        n2 = input.nextInt();
        System.out.println("Escreva o terceiro numero: ");
        n3 = input.nextInt();
        if(n1<n2 && n1<n3){
            if(n2<n3){
                System.out.println("A ordem é: " + n1 + " " + n2 + " " + n3);
            }else {
                System.out.println("A ordem é: " + n1 + " " + n3 + " " + n2);
            }
        }
        if(n2<n1 && n2<n3){
            if(n1<n3){
                System.out.println("A ordem é: " + n2 + " " + n1 + " " + n3);
            }else {
                System.out.println("A ordem é: " + n2 + " " + n3 + " " + n1);
            }
        }
        if(n3<n1 && n3<n2){
            if(n2<n1){
                System.out.println("A ordem é: " + n3 + " " + n2 + " " + n1);
            }else {
                System.out.println("A ordem é: " + n3 + " " + n1 + " " + n2);
            }
        }
        
    }
}
